scripts/pre-commit.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
Can we port this to javascript to make it platform independent ?

Reply:
It is platform independent, it just requires `bash` and `[` :)
I wrote it on Windows with git-bash installed, works for me.

Still, I agree it'd be cool to port to JS, to make it more performant. It matters here, it's `pre-commit`. Bash version spawns git process and node process (eslint) for every staged file, it could be rewitten in Node in such a way that it won't need any extra processes.

Sadly, I don't really have time / willingness to research which library for git is good enough, and how to use it.

We'd need 2 things, obtain list of staged files, and be able to obtain staged version of a file (not the one from working directory).

Reply:
I think we can let the bash script for now, and create another issue for replace it by some je code later.